WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
-                Corr_2018=1.009, CorrSource_2018="Same as 2017",
+        "TTToHadronic_mtop169p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=746.2, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-600To800" : {
-            "CrossSection" : XSValues(
-                XS_2016=12.07, XSSource_2016="XSDB (LO)",
-                XS_2017=12.87, XSSource_2017="XSDB (LO)",
-                XS_2018=12.94, XSSource_2018="GenXSecAnalyzer",
-            ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
-            ),
-            "Correction" : CorrValues(
-                Corr_2017=1.120, CorrSource_2017="https://twiki.cern.ch/twiki/bin/viewauth/CMS/MCKnownIssues#WJetsToLNu_HT_and_DYJets_HT_LO_M",
-                Corr_2018=1.120, CorrSource_2018="Same as 2017",
+        "TTToHadronic_mtop171p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=706.1, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-800To1200" : {
-            "CrossSection" : XSValues(
-                XS_2016=5.497, XSSource_2016="XSDB (LO)",
-                XS_2017=5.366, XSSource_2017="XSDB (LO)",
-                XS_2018=5.451, XSSource_2018="GenXSecAnalyzer",
+        "TTToHadronic_mtop173p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=668.6, XSSource_13TeV="XSDB (NLO)",
             ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+        },
+        "TTToHadronic_mtop175p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=633.4, XSSource_13TeV="XSDB (NLO)",
             ),
-            "Correction" : CorrValues(
-                Corr_2017=1.202, CorrSource_2017="https://twiki.cern.ch/twiki/bin/viewauth/CMS/MCKnownIssues#WJetsToLNu_HT_and_DYJets_HT_LO_M",
-                Corr_2018=1.202, CorrSource_2018="Same as 2017",
+        },
+        "TTToHadronic_mtop178p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=584.6, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-1200To2500" : {
+        "TTToSemiLeptonic" : {
             "CrossSection" : XSValues(
-                XS_2016=1.329, XSSource_2016="XSDB (LO)",
-                XS_2017=1.074, XSSource_2017="XSDB (LO)",
-                XS_2018=1.085, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=687.1, XSSource_13TeV="XSDB (NLO)",
```

Comment:
Please review all of the TTJets samples so that I don't keep making the same comment.